backend/lib/components/Geo/gmaps/getFacDetails.js

```diff
@@ -0,0 +1,50 @@
+const {Client} = require("@googlemaps/google-maps-services-js");
+const Place = require('./utils/place.js');
+const {
+    config: { geo },
+  } = require("../../../../config");
+
+const PLACE_DETAILS_FIELDS = [
+    "formatted_address",
+    "geometry",
+    "url",
+    "adr_address",
+    "business_status",
+    "formatted_phone_number",
+    "international_phone_number",
+    "opening_hours"
+  ];
+
+var placesWithDetails = [];
```

Comment:
Same as above, is there any reason why defining here?

Reply:
Constants are better off sharable for any new exported functions, I think.   I did move the array variables inside the functions, though.   thanks.

Reply:
These are not using anywhere else, let's move this variable inside the method and have it return it in case it is needed anywhere else. 

Also we need to make sure we are not using `var`.

Reply:
Ok - changing all vars to lets here also.